File: CRUD/Java usando NetBeans/SQLite/SistemaSQLite/src/Controlador/Conexion.java
// Written by Ismael Heredia in the year 2016

package Controlador;

import java.sql.Connection;
import java.sql.DriverManager;

public class Conexion {

    public Connection con;

    public void abrir() {
        try {
            Class.forName("org.sqlite.JDBC");
            con = DriverManager.getConnection("jdbc:sqlite:sistema.s3db");
            //con = DriverManager.getConnection("jdbc:sqlserver://localhost;integratedSecurity=true;databaseName=Producciones", "", "");
        } catch (Exception e) {
            System.out.println(e);
        }
    }

    public void cerrar() {
        if (con != null) {
            try {
                con.close();
            } catch (Exception e) {
                //
            }
        }
    }

    public Connection retornar() {
        return con;
    }

}

File: CRUD/C#/Aplicacion con SQLite/aplicacion/Conexion.cs
// Written By Ismael Heredia in the year 2016

using System;
using System.Collections.Generic;
using System.Text;
using System.Data;
using System.Data.SQLite;

namespace aplicacion
{
    class Conexion
    {
        public SQLiteConnection conexion;
        public SQLiteCommand comando;
        public SQLiteDataReader dr;
        public DataTable dt;

        public Conexion()
        {
            conexion = new SQLiteConnection();
            comando = new SQLiteCommand();
            dt = new DataTable();
            dr = null;
        }

        public void abrir()
        {
            conexion.ConnectionString = "Data Source=sistema.s3db;Version=3";
            if (conexion.State == ConnectionState.Open)
            {
                conexion.Close();
            }
            conexion.Open();
            comando.Connection = conexion;
            //comando.CommandType = CommandType.Text;
        }
        public void cerrar()
        {
            conexion.Close();
            //conexion.Dispose();
        }
    }
}
